src/ios/CDVPurchasesPlugin.m

```diff
@@ -29,7 +29,8 @@ - (void)setupPurchases:(CDVInvokedUrlCommand *)command {
                         observerMode:observerMode
                userDefaultsSuiteName:userDefaultsSuiteName
                       platformFlavor:self.platformFlavor
-               platformFlavorVersion:self.platformFlavorVersion];
+               platformFlavorVersion:self.platformFlavorVersion
+                   dangerousSettings:nil];
```

Comment:
this is the breaking change from hybridcommon 2.0

Reply:
Hmm looks like this was done in iOS only. I believe this is correct but just to confirm, android didn't have that change then right?

Reply:
It's iOS only. I don't quite recall _why_ 😅 

https://github.com/RevenueCat/purchases-hybrid-common/releases/2.0.0

https://github.com/RevenueCat/purchases-hybrid-common/blob/2.0.0/android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt#L320

Reply:
it was added in 3.0.0
https://github.com/RevenueCat/purchases-hybrid-common/compare/2.0.0...3.0.0#diff-77d224542d5cfe0397ffb1d5cd7af37a219d8587772ed4d63d777ab3d0f0bc91R294

we had to update to the next major for iOS and Android as separate tasks, I think that's why we did this